cmd/addvalidators.go

```diff
@@ -33,9 +37,10 @@ type addValidatorsConfig struct {
 	WithdrawalAddrs   []string // Withdrawal address of each validator
 	FeeRecipientAddrs []string // Fee recipient address of each validator
 
-	Lockfile        string   // Path to the legacy cluster lock file
-	ManifestFile    string   // Path to the cluster manifest file
-	EnrPrivKeyfiles []string // Paths to node enr private keys
+	Lockfile          string   // Path to the legacy cluster lock file
+	ManifestFile      string   // Path to the cluster manifest file
+	ManifestBackupDir string   // Directory to store cluster manifest backups
```

Comment:
I think we can follows Oisin suggestion to not create a seperate directory for backups.